src/plugins/auto/plugin.cpp

```diff
@@ -329,18 +329,22 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
     // updateFromMap will check config valid
     loadConfig.UpdateFromMap(config, GetName(), true);
     auto fullConfig = loadConfig._keyConfigMap;
+    bool workModeAuto = GetName() == "AUTO";
     // Remove the performance hint if no setting to this property from user.
     if (!loadConfig._isSetPerHint) {
```

Comment:
Need to check if Hint is set in config

Reply:
The above code at line 330 has been checked if performance hint is set.
`loadConfig.UpdateFromMap(config, GetName(), true);`